zer0-os/zOS
# Update the PWA manifest and icons

### What does this do?

This updates the PWA manifest name and updates the icons to not be the react icon.

### Why are we making this change?

When creating a desktop shortcut these names and icons are used and we don't want the react app defaults :)

### How do I test this?

Load up the browser ande try to create a desktop shortcut and verify the the icon and name are defaulted correctly.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
